Config.uk

```diff
@@ -86,6 +86,15 @@ config OPTIMIZE_LTO
 		will increase overall building time but creates more efficient
 		Unikraft binaries.
 
+config ENABLE_PIE
```

Comment:
Hey, can you call this option OPTIMIZE_PIE ? This would be inline with the other optimization flags and could be catched by our build script that watches changes of compiler optimization flags.